x-pack/plugins/rule_registry/server/utils/create_lifecycle_rule_type.test.ts

```diff
@@ -188,44 +188,6 @@ describe('createLifecycleRuleTypeFactory', () => {
 
         expect(documents.map((doc) => omit(doc, 'kibana.rac.alert.uuid'))).toMatchInlineSnapshot(`
           Array [
-            Object {
-              "@timestamp": "2021-06-16T09:01:00.000Z",
-              "event.action": "open",
```

Comment:
More like a general comment. `event.action` is basically the status of the document, right? Action sounds confusing at least to me.

Reply:
I believe this is ECS defined: https://www.elastic.co/guide/en/ecs/current/ecs-event.html

Reply:
Ok thanks makes more sense now. Felix wrote a test as part of this PR and looks like event.action is different from event.status.